real_working_app.py

```
@@ -0,0 +1,614 @@
+        country_domains = {
+            'SA': 'sa', 'AE': 'ae', 'EG': 'eg', 'KW': 'kw', 'QA': 'qa',
+            'BH': 'bh', 'OM': 'om', 'JO': 'jo', 'LB': 'lb', 'SY': 'sy',
+            'IQ': 'iq', 'YE': 'ye', 'SD': 'sd', 'LY': 'ly', 'TN': 'tn',
+            'DZ': 'dz', 'MA': 'ma'
+        }
+        
+        domain = country_domains.get(country, 'com')
+        email = f"{username}@temp.{domain}"
+        password = hashlib.md5(f"{username}{timestamp}".encode()).hexdigest()[:8]
+        
+        print(f"✅ تم إنشاء البريد المؤقت بنجاح!")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)